nimpretty/tests/expected/exhaustive.nim

```diff
@@ -122,7 +121,7 @@ type
     inquote {.pragmaHereWrongCurlyEnd.}: bool
     col, lastLineNumber, lineSpan, indentLevel: int
     content: string
-    fixedUntil: int # marks where we must not go in the content
+    fixedUntil: int                    # marks where we must not go in the content
```

Comment:
This looks like a bug to me...

Reply:
This is part of the same `object` section like the field and its comment on the line below — it aligns both those comments.

The reason why it wasn't aligned already is because this way the line is 82 characters long. Had the comment been just a slightly shorter, this align would have already been there all this time.